src/components/NextFab.tsx

```diff
@@ -0,0 +1,25 @@
+import Fab from '@mui/material/Fab';
```

Comment:
I think NextFab and PreviousFab should be named according to their purpose, not their implementation.  Although they happen to be floating action buttons now, they might have a different implementation in the future.  Likewise, other developers might not know what "NextFab" is for, but "NextChapterButton" is crystal-clear.